### PR TITLE
added signed response example + logging headers example

### PR DESCRIPTION
**Pull request by davidtsong**

### CODE REVIEW

File: src/content/examples/logging-headers.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+order: 1000
+type: example
+summary: Examine the contents of a Headers object by logging to console with a Map.
+demo: https://returning-json.workers-sites-examples.workers.dev
+tags:
+  - Middleware
+  - Originless
+---
+
+# Logging Headers
```
Comment (adamschwartz) [inline src/content/examples/logging-headers.md:11]: Lowercase “h”.

File: src/content/examples/logging-headers.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+console.log(new Map(request.headers))
+```
+
+Use the spread operator if you need to quickly stringify a Headers object:
+
+```javascript
+let requestHeaders = JSON.stringify([...request.headers])
+```
+
+### The Problem
```
Comment (adamschwartz) [inline src/content/examples/logging-headers.md:54]: Lowercased “p”.

File: src/content/examples/logging-headers.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+When debugging Worker scripts, we often want to examine the headers on a request
+or response. A common pitfall is to try to log headers to the developer console
+via code like this:
+
+```javascript
```
Comment (adamschwartz) [inline src/content/examples/logging-headers.md:60]: `js`.

File: src/content/examples/logging-headers.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+console.log(request.headers)
+```
+
+or this:
+
+```javascript
```
Comment (adamschwartz) [inline src/content/examples/logging-headers.md:66]: `js`.

File: src/content/examples/logging-headers.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+Headers objects are iterable, however, which we can take advantage of to develop
+a couple quick one-liners for debug-printing headers.
+
+## Pass Headers objects through a Map
+
+The first common idiom for making Headers `console.log()`-friendly is to
+construct a Map object from the Headers object, and log the Map object.
+
+```javascript
```
Comment (adamschwartz) [inline src/content/examples/logging-headers.md:88]: `js`.

File: src/content/examples/logging-headers.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+While not as elegant as object literal syntax, this is certainly readable and
+useful for debugging purposes.
+
+
+
+<!-- ## Demo
+
+<p><a href={props.frontmatter.demo}>Open demo</a></p>
+
+<Demo src={props.frontmatter.demo} title={props.frontmatter.summary} height="80"/> -->
```
Comment (adamschwartz) [inline src/content/examples/logging-headers.md:155]: Remove this commented out demo (and whitespace).

File: src/content/examples/signed-request.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+  console.log(createHexSignature("asd"))
+  event.respondWith(handleRequest(event.request))
+})
+```
+
+<!-- ## Demo
+
+<p><a href={props.frontmatter.demo}>Open demo</a></p>
+
+<Demo src={props.frontmatter.demo} title={props.frontmatter.summary} height="80"/> -->
```
Comment (adamschwartz) [inline src/content/examples/signed-request.md:65]: Remove this commented out demo.

**Comment by adamschwartz** [comment]
[`e75b9f0` (#148)](https://github.com/cloudflare/workers-docs-engine/pull/148/commits/e75b9f0bf5229f196e155853ed67486bf36ba3af) This seems like a mistake.

**Comment by davidtsong** [comment]
> [`e75b9f0` (#148)](https://github.com/cloudflare/workers-docs-engine/pull/148/commits/e75b9f0bf5229f196e155853ed67486bf36ba3af) This seems like a mistake.

Removed this plugin because Ian mentioned it doesn't work properly for # mdx links. Should install and create a PR for it?

**Comment by davidtsong** [comment]
I think this should be ready to merge now

**Pull request closed without merging**